ganga/GangaCore/GPIDev/Lib/Job/Job.py

```diff
@@ -881,15 +881,19 @@ def createPackedInputSandbox(self, files, master=False):
         """
 
         name = '_input_sandbox_' + self.getFQID('_') + '%s.tgz'
-
         if master:
             if self.master is not None:
                 name = '_input_sandbox_' + self.master.getFQID('_') + '%s.tgz'
             name = name % "_master"
         else:
             name = name % ""
 
-        files = [f for f in files if hasattr(f, 'name') and not f.name.startswith('.nfs')]
```

Comment:
I'm not sure why this line had to be un-condensed but files was empty if now when making the master sandbox. I think it has something to do with `files` being a `map`.